cohorts/varcode_utils.py

```diff
@@ -11,7 +11,12 @@ def __init__(self, variant, variant_collection, patient):
 
     @property
     def variant_metadata(self):
-        return self.variant_collection.metadata[self.variant]
+        source_to_metadata = self.variant_collection.source_to_metadata_dict
```

Comment:
I'm still a bit confused: it looks like before we had a single variant metadata dict returned here, and now it's a dict from source to variant-metadata?